scripts/test_mail-alarm.py

```diff
@@ -25,6 +26,24 @@ def mock_setup(module):
     module.branding.PRODUCT_BRAND = "XenServer"
     module.branding.BRAND_CONSOLE = "XenCenter"
 
+def get_alarm_xml(xmlalarm_str, xmlbody_str, xmlcls_str, xmlname_str):
```

Comment:
Ignore previous message. I tried in python and it will work even if given placeholder is missing.
Thanks!